cuda_bindings/cuda/bindings/_path_finder/load_nvidia_dynamic_library.py

```diff
@@ -38,6 +59,15 @@ def _load_nvidia_dynamic_library_no_cache(libname: str) -> LoadedDL:
         loaded = load_with_system_search(libname, found.lib_searched_for)
         if loaded is not None:
             return loaded
+        if libname == "nvvm":
+            # Use cudart as anchor point (libcudart.so.12 is only ~720K, cudart64_12.dll ~560K).
```

Comment:
`cudart` is probably not the best choice here as it's by default statically linked by `nvcc` and is most commonly statically linked.

Reply:
Changed to `nppc` as `ANCHOR_LIBNAME`: commit 2a039d2a7ec44e1d77b68b7fcc610897fcb4bd4f

That library is smaller than `cudart` on Windows, but more than double the size of `cudart` on Linux. But `nppc` will only be imported once per process, and only if `nvvm` is requested.